example/src/main/java/com/stripe/example/service/BackendApi.kt

```diff
@@ -21,5 +21,5 @@ interface BackendApi {
 
     @FormUrlEncoded
     @POST("create_setup_intent")
-    fun createSetupIntent(@FieldMap params: HashMap<String, Any>): Observable<ResponseBody>
+    fun createSetupIntent(@FieldMap params: MutableMap<String, Any>): Observable<ResponseBody>
```

Comment:
Why does this need to be mutable (or more specifically, why can't this just be `Map`)?

Reply:
This is some limitation in OkHttp, can't remember the exact reason. I made it a MutableMap to make it more generic than HashMap.